### DISCUSSION TITLE
Where can I get the code for the Advanced Example

### DISCUSSION THREAD

**Discussion by xiaozhen123321** [videojs/video.js#8383] [Q&A]
Where can I get the code of the demo linked below
[https://videojs.com/advanced?video=sintel](https://videojs.com/advanced?video=sintel)

**Comment by amtins** [comment]
Hi @lizhen12345 you should be able to find everything here https://github.com/videojs/videojs.com/tree/main/src/components/HomeComponents/AdvancedExample

**Reply by xiaozhen123321** [reply]
tks very much
